test/request_quota_increase_test.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestRequestQuotaIncrease(t *testing.T) {
+	t.Parallel()
+
+	awsRegion := aws.GetRandomRegion(t, nil, nil)
+
+	terraformOptions := &terraform.Options{
+		TerraformDir: "../examples/request-quota-increase",
+		Vars: map[string]interface{}{
+			"aws_region": awsRegion,
+		},
+	}
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApply(t, terraformOptions)
+
+	newQuotas := terraform.OutputMapOfObjects(t, terraformOptions, "new_quotas")
+
+	natGateway := newQuotas["nat_gateway"]
+	natGatewayMap := natGateway.(map[string]interface{})
+
+	assert.Equal(t, natGatewayMap["quota_name"], "NAT gateways per Availability Zone")
+	assert.Equal(t, natGatewayMap["service_name"], "Amazon Virtual Private Cloud (Amazon VPC)")
+
+	naclRules := newQuotas["nacl_rules"]
+	naclRulesMap := naclRules.(map[string]interface{})
+
+	assert.Equal(t, naclRulesMap["quota_name"], "Rules per network ACL")
+	assert.Equal(t, naclRulesMap["service_name"], "Amazon Virtual Private Cloud (Amazon VPC)")
```

Comment:
Thanks for adding the test!

Note that there are many opportunities for panics here: e.g., if the output format changes a bit. The panic would prevent cleanup of our tests, which wouldn't be great. 

I'd recommend defining a `struct` that represents the output you expect and using [`OutputStruct`](https://github.com/gruntwork-io/terratest/blob/master/modules/terraform/output.go#L289) to parse the output into that struct, or return an error (but not panic!) if anything is missing.

Reply:
> Note that there are many opportunities for panics here: e.g., if the output format changes a bit. 

Good to know!